test/webpack/basic-test.js

```diff
@@ -36,6 +36,29 @@ describe("iconv-lite", function() {
         var str = iconv.decode(buf, "utf8");
         assert.equal(str, "💩");
     });
+
+    it("supports passing Uint8Array to decode for all encodings", function() {
+        iconv.encode('', 'utf8'); // Load all encodings.
+
+        const encodings = Object.keys(iconv.encodings)
+        encodings.forEach(function(encoding) {
+            if (['base64', 'hex', '_internal', '0'].indexOf(encoding) >= 0) {
```

Comment:
Excluded temporarily `base64` and `hex` since they don't seem to work even with buffers: https://github.com/ashtuchkin/iconv-lite/issues/247